src/mito2/src/region.rs

```diff
@@ -526,6 +522,18 @@ impl OpeningRegions {
 
 pub(crate) type OpeningRegionsRef = Arc<OpeningRegions>;
 
+/// Single region stats.
+#[derive(Default, Debug, Clone)]
+pub(crate) struct Stats {
+    total_manifest_size: Arc<AtomicU64>,
```

Comment:
Looks like the total data size for the single region is missing.

Reply:
The `Stats` stores the single region statistics.

Reply:
Yep, it only contains the manifest size, can it include the data size too?

Reply:
I got you, we can introduce it in the future. Let's fix the bug first.